tests/e2e/specs/wcpay/shopper/shopper-checkout-stripe-link.spec.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { merchantWCP, shopperWCP } from '../../../utils/flows';
+import { randomizeEmail, getInputTextValue } from '../../../utils';
+import { fillCardDetails, setupProductCheckout } from '../../../utils/payments';
+const { shopper, merchant, uiUnblocked } = require( '@woocommerce/e2e-utils' );
+
+const linkPaymentsCheckbox = '#inspector-checkbox-control-10';
+
+describe( 'Link with enabled UPE', () => {
```

Comment:
This class has tests for both UPE & split UPE because they need to be executed sequentially. Parallel execution at the same time leads to `429 Too Many Requests`.